testing/scripts/Makefile

```diff
@@ -18,14 +18,10 @@ kind_build_operator:
 
 kind_build_images: kind_build_engine kind_build_operator
 
-install_helm:
-	kubectl -n kube-system create sa tiller
-	kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount=kube-system:tiller
-	helm init --service-account tiller
-	kubectl rollout status deploy/tiller-deploy -n kube-system
-
 install_ambassador:
-	helm install stable/ambassador -f ambassador_values.yaml --name ambassador --set crds.keep=false --namespace seldon --set replicaCount=1
+	helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+	helm repo update
+	helm install ambassador stable/ambassador -f ambassador_values.yaml --set crds.keep=false --namespace seldon --set replicaCount=1
```

Comment:
Right now the end to end tests use the dependencies in the core-builder image, which I believe don't use helm 3.x, so this change would not only have to be done in the core-builder image, and you would have to change the image that it points towards